server/lib/wsapi/lobbies.js

```diff
@@ -160,7 +160,16 @@ export class LobbyApi {
     // Team Melee and FFA always provide 8 player slots, divided amongst the teams evenly
     const numSlots = gameType === 'teamMelee' || gameType === 'teamFfa' ? 8 : mapData.slots
 
-    const lobby = Lobbies.create(name, mapData, gameType, gameSubType, numSlots, client.name)
+    const lobby = Lobbies.create(
+      name,
+      mapData,
+      gameType,
+      gameSubType,
+      numSlots,
+      client.name,
+      undefined,
```

Comment:
put a comment for which parameter this is, i.e.:
`undefined, /* hostRace */`

Probably at this point we're getting to where a params object makes more sense than an ordered list, though :)